src/redis_client.py

```diff
@@ -129,8 +134,9 @@ def write_summary_redis(redis_db, experiment_iteration_count, mimr, perf_gain, a
     redis_db.hset(hash_name, 'perf_improvement', perf_gain)
     redis_db.hset(hash_name, 'action_taken', action_taken_str)
     redis_db.hset(hash_name, 'current_perf', current_perf)
-    redis_db.hset(hash_name, 'elapsed_time', elaps_time)
+    redis_db.hset(hash_name, 'elapsed_time', elapsed_time)
     redis_db.hset(hash_name, 'cumulative_mr', cumulative_mr)
```

Comment:
The parameter has been changed to cumm_mr. It should read redis_db.hset(hash_name, 'cumulative_mr', cumm_mr)